src/main/java/Config/Config.java

```diff
@@ -49,9 +50,20 @@ private static Routes setRoutes() {
     routes.add(new Route(GET, "/redirect", new RedirectHandler()));
     routes.add(new Route(GET, "/logs", new AUTHHandler()));
     routes.add(new Route(GET, "/parameters", new ParameterHandler()));
+    setStaticRoutes(routes);
     return routes;
   }
 
+  private static void setStaticRoutes(Routes routes) {
+    Directory directory = new Directory(publicDirectory);
+    for (String fileName : directory.getFileNames()) {
```

Comment:
I really like the way that you're looping through your files and adding routes for each one. Which story are you adding the POST, PUT, and DELETE file routes for? Besides for log files and other utility files, it's pretty uncommon to update or delete a file from a server route.

Reply:
Ahh I think I got this mixed up then. Should I maybe create a storage object, rather than a file?

Reply:
I'd say go with a storage object, memory storage is probably fine - instead of saving it forever as a file, this way you could just have it store things in an internal hash map (or whatever).